static/js/components/CommentTree.js

```diff
@@ -120,7 +120,7 @@ export default class CommentTree extends React.Component<*, *> {
               upvote={upvote}
               downvote={downvote}
             />
-            {atMaxDepth || moderationUI
+            {atMaxDepth || moderationUI || comment.deleted
```

Comment:
might be good to add a quick test to check for this, I think there's one already that checks that `moderationUI` hides this button, maybe could just make than one into a parameterized test or something.